src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -2638,7 +2638,7 @@ private bool MemberGroupFinalValidation(BoundExpression? receiverOpt, MethodSymb
             // applicable candidate set, so the applicable candidate set consists solely of
             // M(object, object) and is therefore the best match.
 
-            return !methodSymbol.CheckConstraints(new ConstraintsHelper.CheckConstraintsArgs(this.Compilation, this.Conversions, includeNullability: false, node.Location, diagnostics));
+            return !methodSymbol.CheckConstraintsIncludingExtension(new ConstraintsHelper.CheckConstraintsArgs(this.Compilation, this.Conversions, includeNullability: false, node.Location, diagnostics));
```

Comment:
>CheckConstraintsIncludingExtension [](http://example.com/codeflow?start=33&length=34)

Is this change covered by a test?

Reply:
Will undo this change and the one in `GetCandidatesPassingFinalValidation`. In the language versions where we include new extension members in overload resolution, we filter bad candidates (which fail constraint checks) early, so the final validation check doesn't make a difference.